### ISSUE TITLE
toggle staking for specific transactions

### ISSUE THREAD

**Issue by mdpfeiffer** [PaycoinFoundation/paycoin#299]
Proposal:
In “coin control” allow users to toggle staking for specific transactions. 

Rationale:
Now that users have microprimes, con control features need to be more integrated, accessible, and discoverable. If a user is staking coins on a microprime and forgets or fails to use coin control, they will likely send coins that have been accruing coinday, disrupting their staking plan, and they may not even realize it until the next time they check their wallet. For casual and novice users, that could be a long time and lead to frustration and disappointment.

I know that there’s already a thread to "Add a '-stake' flag to disable the staking thread #268," but I think that may solve a different problem.

https://github.com/PaycoinFoundation/paycoin/pull/268

**Comment by IngCr3at1on** [comment]
Yeah #268 is to **disable** staking entirely in a wallet not for a specific address.
